src/gui/SettingsWindow.h

```diff
@@ -18,6 +20,10 @@ class SettingsWindow : public QDialog {
     void closeEvent(QCloseEvent *event) override;
     void keyPressEvent(QKeyEvent *event) override;
     void keyReleaseEvent(QKeyEvent *event) override;
+    void dragEnterEvent(QDragEnterEvent *event) override;
+    void dragLeaveEvent(QDragLeaveEvent *event) Q_DECL_OVERRIDE;
```

Comment:
why does this use Q_DECL_OVERRIDE?

Reply:
`Q_DECL_OVERRIDE` is a preprocessor macro that expands to the C++11 keyword "override" if the project is built with the C++11 options on and the compiler supports it, or nothing otherwise.

Reply:
as we do not care about pre C++11 (or actually C++17), changed it to `override`.